### ISSUE TITLE
NullReferenceException with RealAntennas

### ISSUE THREAD

**Issue by SirMortimer** [ChrisAdderley/NearFutureExploration#18]
This is log spammed in the editor when placing a communotron-16 on a vessel.

```
[EXC 21:27:18.842] NullReferenceException: Object reference not set to an instance of an object
        NearFutureExploration.ModuleAntennaFeed.FindAntenna () (at <674e7ceace814edc8162c68718264bea>:0)
        NearFutureExploration.ModuleAntennaFeed.Start () (at <674e7ceace814edc8162c68718264bea>:0)
        UnityEngine.DebugLogHandler:LogException(Exception, Object)
        ModuleManager.UnityLogHandle.InterceptLogHandler:LogException(Exception, Object)
        UnityEngine.Debug:CallOverridenDebugHandler(Exception, Object)
...
[EXC 21:27:19.954] NullReferenceException: Object reference not set to an instance of an object
        NearFutureExploration.ModuleAntennaFeed.NullReflectorBonus () (at <674e7ceace814edc8162c68718264bea>:0)
        NearFutureExploration.ModuleAntennaFeed.FixedUpdate () (at <674e7ceace814edc8162c68718264bea>:0)
        UnityEngine.DebugLogHandler:LogException(Exception, Object)
        ModuleManager.UnityLogHandle.InterceptLogHandler:LogException(Exception, Object)
        UnityEngine.Debug:CallOverridenDebugHandler(Exception, Object)
```

Other antenna-related mod installed: RealAntennas.
The vessel was loaded from an existing file, and did not previously have this issue.

**Comment by ChrisAdderley** [comment]
Will be added to exclusion list in 1.0.3.

**Comment by ChrisAdderley** [comment]
This should be fixed in 1.0.3 now.

**Comment by SirMortimer** [comment]
thanks!
